plugin/doc_comment.js

```diff
@@ -123,12 +123,15 @@
     var result = comments[comments.length - 1];
     if (cx.parent._docComment.fullDocs) {
       result = result.trim().replace(/\n[ \t]*\* ?/g, "\n");
+      result = result.replace(/^\*/g, "");
     } else {
       var dot = result.search(/\.\s/);
       if (dot > 5) result = result.slice(0, dot + 1);
-      result = result.trim().replace(/\s*\n\s*\*\s*|\s{1,}/g, " ");
+      // Remove all space*space in beginning of the lines
+      result = result.replace(/^\n?([ \t]*\* *)/gm, "");
```

Comment:
This is going to match at most one time (since it includes `^`). That seems unlikely to be what you intended.